client/src/modules/vx-timer/store/mutations.ts

```diff
@@ -57,13 +58,13 @@ export default {
     const { timers } = state
     if (!id) throw new Error(`[Error] Timer ID was not provided`)
     if (!timers.hasOwnProperty(id)) throw new Error(`[Error] Timer with ID [${id}] does not exist`)
+    // console.info('Refreshing timer =', id) // eslint-disable-line no-console
```

Comment:
Intentional comment?

Reply:
Yes. I used these console.info()s to verify the timer behaviours... But atm we're only used the 1 timer (and I think it's OK now). When we enable the other timers, we may need to verify their behaviours too...